### DISCUSSION TITLE
What is "Recommended" label in GitHub marketplace?

### DISCUSSION THREAD

**Discussion by elv1s42** [community/community#55440] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

In the GitHub Marketplace, there are some of GitHub applications that are marked with `Recommended` label.
What are the criteria for the application to become `Recommended`?
There is no information about this topic in GitHub documentation.

**Comment by parthgupta1208** [comment]
# The "Recommended" label is used to indicate applications that GitHub believes are valuable and high-quality options for developers.

The criteria for an application to become "Recommended" may include:

- Quality and usefulness: Recommended applications are expected to provide high-quality functionality and offer valuable features to GitHub users. They should demonstrate a clear purpose and address specific developer needs.

- Positive user feedback: Applications that receive positive reviews and feedback from users are more likely to be considered for the "Recommended" label. GitHub takes into account the experiences and satisfaction of existing users when determining recommendations.

- Adoption and popularity: Applications that have a significant number of active installations and a growing user base may have a higher chance of being recommended. GitHub considers the overall popularity and adoption of an application as an indicator of its usefulness and value to the community.

- Integration with GitHub: Applications that integrate seamlessly with GitHub and provide a smooth user experience are more likely to be recommended. They should leverage GitHub's APIs and platform features effectively to enhance the development workflow.

- Compliance with GitHub policies: Applications must comply with GitHub's terms of service, policies, and guidelines to be eligible for the "Recommended" label. They should meet the required standards for security, privacy, and data handling.

#### It's important to note that the specific criteria and evaluation process for the "Recommended" label may be subject to change by GitHub.

**Reply by elv1s42** [reply]
This is a very good answer, thank you.
However, it looks like these rules are not working, also they are not documented anywhere, which is confusing.

Here is an example, where a deprecated application is marked with `recommended` label:
![image](https://github.com/community/community/assets/6689884/b28c04f9-65ec-4199-9956-356396072f4d)
